### DISCUSSION TITLE
Formatting of extracted text with indentation

### DISCUSSION THREAD

**Discussion by TheEyesChico** [jsvine/pdfplumber#442] [Ask for help with specific PDFs]
<h3> Hey guys, finding your utility very useful so kudos to that! </h3>

Okay so I'm working on a high level project where I need to put the data from pdf inside dataframes division by division (for extraction purpose). Indentation in this process becomes important because if I'm searching a particular keyword in a document, I need to back-track the indentation from sub sections to top sections.

- <h4> PDF Data </h4>

![image](https://user-images.githubusercontent.com/64800839/120277469-b7a61380-c2d1-11eb-95dc-b198560581c9.png)

However the output I get is very structured and indentation usually isn't taken care of.

- <h4> Output </h4>

![image](https://user-images.githubusercontent.com/64800839/120279657-53d11a00-c2d4-11eb-8e70-a49be2db3e5a.png)

**Comment by samkit-jain** [comment]
Hi @TheEyesChico Appreciate your interest in the library. This functionality is not yet supported but I can see how it will be useful to you and many others. There's already an open issue https://github.com/jsvine/pdfplumber/issues/10 for tracking this. Request you to follow that. You may also contribute to the development if you have a solution.
